prow/jobs/console/console-integration.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-project/console
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/console/tests"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository: "true"
+
+presubmits: # runs on PRs
+  kyma-project/console:
+    - name: console-integration
```

Comment:
Rename to `console-tests` and move it to `prow/jobs/console/tests`, and rename file to `tests.yaml` or `console-tests.yaml` as `tests` is generic name (file names in `jobs` directory must be unique).

Naming convention is described here: https://github.com/kyma-project/test-infra/blob/master/prow/README.md#convention-for-naming-jobs
Structure of `jobs` directory: https://github.com/kyma-project/test-infra/blob/master/prow/README.md#strategy-for-organizing-jobs
Also as you can see in other jobs, we don't use a special name for jobs that build components.

Reply:
done, thanks for the feedback